brushfire-scalding/src/main/scala/com/stripe/brushfire/scalding/Trainer.scala

```diff
@@ -222,7 +223,7 @@ case class Trainer[K: Ordering, V, T: Monoid](
    * @tparam E
    * @return
    */
-  def featureImportance[P, E](error: Error[T, P, E])(fn: TypedPipe[(K, E)] => Execution[_])(implicit voter: Voter[T, P]) = {
+  def featureImportance[P, E](error: Error[T, P, E])(fn: TypedPipe[(K, E)] => Execution[A])(implicit voter: Voter[T, P]) = {
```

Comment:
`Execution[A]` fixes the typechecker but feels a bit odd. I'm guessing something else is broken.